1-exercises/B-while-loop/exercise.js

```diff
@@ -6,9 +6,18 @@
 */
 
 function evenNumbers(n) {
-    // TODO
+    let count = 0;
+    let result = [];
+    if (n) {
+        while (count < n) {
+            count === 0 ? result.push(count) : result.push(count * 2);
```

Comment:
This works well - you don't need to do the check, you can just use result.push(count * 2) as 0x2=0.